src/App.tsx

```diff
@@ -41,6 +41,12 @@ function App() {
     setTasks((prevTasks) => prevTasks.filter((task) => task.id !== taskId));
   };
 
+  const handleTaskUpdate = (updatedTask: Task) => {
+    setTasks((prevTasks) =>
+      prevTasks.map((task) => (task.id === updatedTask.id ? updatedTask : task))
+    );
+  };
```

Comment:
я бы сократил возможность до обновления только вэлью.

```suggestion
  const handleTaskUpdate = (id: string, value: string) => {
    ...
  };
```

Reply:
я бы ещё унифицировал названия. У тебя есть `handleComplete` `handleTaskUpdate` `handleAddTask`
надо выбрать какой-то один способ написания типовых действий